server/Makefile

```
@@ -767,20 +767,19 @@ ifeq ($(BUILD_ENTERPRISE_READY),true)
 endif
 
 vet: ## Run mattermost go vet specific checks
-	$(GO) install github.com/mattermost/mattermost-govet/v2@new
-	@VET_CMD="-license -structuredLogging -inconsistentReceiverName -inconsistentReceiverName.ignore=session_serial_gen.go,team_member_serial_gen.go,user_serial_gen.go -emptyStrCmp -tFatal -configtelemetry -errorAssertions"; \
-	if ! [ -z "${MM_VET_OPENSPEC_PATH}" ] && [ -f "${MM_VET_OPENSPEC_PATH}" ]; then \
-		VET_CMD="$$VET_CMD -openApiSync -openApiSync.spec=$$MM_VET_OPENSPEC_PATH"; \
-	else \
-		echo "MM_VET_OPENSPEC_PATH not set or spec yaml path in it is incorrect. Skipping API check"; \
-	fi; \
-	$(GO) vet -vettool=$(GOBIN)/mattermost-govet $$VET_CMD ./...
+	$(GO) install github.com/mattermost/mattermost-govet@6c71c4bcb26283d06aa6c5e2dcb3a2cdfe15b3be
```

I think this needs to be as following:
```suggestion
	$(GO) install github.com/mattermost/mattermost-govet/v2@4dfa609fd5202ce3982a141a98f8116bb5bd2721
```
I got an error about the commit couldn't be found. Maybe some commits have been changed after you push?

---

Possibly, yes! I'm hoping to remove the commit hash altogether merge https://github.com/mattermost/mattermost-govet/pull/36.